descriptors/ModuleDescriptor-template.json

```diff
@@ -41,6 +41,17 @@
         }
       ]
     },
+    {
+      "id": "anonymize-loans",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/anonymize-loans",
```

Comment:
I thought we agreed that the path would be storage specific, in order to reserve a top-level path like `/anonymize-loans` for a business logic endpoint, like manual anonymization.

Can you help me understand why this path was chosen? Was it to do with the RAML Module Builder limitations of overlapping paths?

Reply:
It has to do with the RMB issue. It looks like I cant add a POST /loan-storage/loans/anonymize endpoint if there is already POST /loan-storage/loans/anonymize{userId} defined

Which endpoint would you suggest to use?

Reply:
Please can you raise a bug report for this in the RAML Module Builder project, using these example endpoints

I suggest using any path that includes storage and doesn't cause this conflict, for example, `anonymize-storage-loans`

This can easily be changed as the interface is marked experimental.

Reply:
The interface has been renamed to /anonymize-storage-loans